frappe/desk/page/applications/applications.py

```diff
@@ -25,12 +30,31 @@ def get_app_list():
 		if app in installed:
 			out[app]["installed"] = 1
 
+	contrib = get_contrib_apps()
+	contrib = {app:contrib[app] for app in contrib if app not in installed}
+	out.update(contrib)
 	return out
 
+def get_contrib_apps():
+	contrib_apps_url = 'https://raw.githubusercontent.com/frappe/bench/master/install_scripts/contrib-apps.json'
+	return requests.get(contrib_apps_url).json()
```

Comment:
@pdvyas lets add contrib apps in frappe and not do a requests, as we will have to add pyssl for this.

Will you fix this?